recipes/inference-server/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "inference-server" %}
+{% set version = "1.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/inference_server-{{ version }}.tar.gz
+  sha256: 08bc663404dbddb160b7cd8cf9f507b0baed8cd3271d66baf11c1f1f939a58ac
+
+build:
+  skip: true  # [py<38]
```

Comment:
Looks like this package can be noarch, right?

https://conda-forge.org/docs/maintainer/knowledge_base/#noarch-python